examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -332,7 +332,7 @@ def train_transforms(batch):
                 audio["array"], max_length=data_args.max_length_seconds, sample_rate=feature_extractor.sampling_rate
             )
             subsampled_wavs.append(wav)
-        inputs = feature_extractor(subsampled_wavs, sampling_rate=feature_extractor.sampling_rate)
+        inputs = feature_extractor(subsampled_wavs, sampling_rate=feature_extractor.sampling_rate, max_length=16000, truncation=True, padding=True)
```

Comment:
Does running on A100 require the same change? If not, this issue should also be reported.

Reply:
A100 requires the same change to run normally so I think this is not the maf issues.